src/HasValidationMessage.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I don't sure that this trait is really useful. Maybe we can place the `message` method into parent `Rule` instead?

Reply:
Yes, we can. But not all validators will use it. So I put it in the trait.

Reply:
Which ones?

Reply:
Callback, CompareTo, Ip, Number

Reply:
Ok thanks